src/modules/dashboard/sagas/colonyCreate.ts

```diff
@@ -153,10 +155,16 @@ function* colonyCreate({
     });
 
     if (createToken) {
+      const { address: tokenLockingAddress } = yield call([
+        networkClient.getTokenLockingAddress,
+        networkClient.getTokenLockingAddress.call,
```

Comment:
Could this fail? Could this be not defined? Just making sure we're handling all the cases.

Reply:
I don't think It can be `undefined`, unless colonyJS crashes completely.

It can however fail. I'm thinking infura connectivity troubles. But if that's the case the whole creation process fails, since everything is wrapped in big try/catch block.

The only alternative I can think of, is re-wrapping just this in another try/catch block, and if it fails, just _silently_, leave `allowToTransfer` as a empty `[]` array. But that takes us back to our original issue.